src/manage_nsfs/manage_nsfs_help_utils.js

```diff
@@ -281,6 +281,42 @@ Usage:
 
 `;
 
+
+const UPGRADE_OPTIONS = `
+Usage:
+
+'upgrade' is a noobaa-core command that will return the exported metrics of the deployed NooBaa system.
+
+Usage:
+
+    noobaa-cli upgrade <action> [flags]
+
+List of actions supported:
+
+    start
+    status
+    history
+    
+`;
+
+const UPGRADE_START_OPTIONS = `
+'upgrade start' is a noobaa-core command that will start config directory upgrade run.
+Run upgrade start after upgrading NooBaa RPMs on all the cluster nodes, after starting an upgrade of the config directory,
+S3 I/O, S3 Buckets getters and NooBaa CLI Account/Buckets/Whitelist getters operations will still be working
+But updates of the config directory will be blocked during the upgrade of the config directory.
+`;
+
+const UPGRADE_STATUS_OPTIONS = `
+'upgrade status' is a noobaa-core command that will return the status of an ongoing upgrade run.
+
+`;
+
+const UPGRADE_HISTORY_OPTIONS = `
+'upgrade history' is a noobaa-core command that will return the history of past upgrades.
```

Comment:
Can you give more details about it? (What can we see in the upgrade history?)

Reply:
it'll be finalized in the next PR but I added some and we can change it later if needed